extension/src/validator/validator-builder.js

```diff
@@ -20,6 +21,23 @@ function withPayment (paymentObject) {
           = errorMessages.SUBMIT_ADDITIONAL_PAYMENT_DETAILS_REQUEST_INVALID_JSON
       return this
     },
+    validateAmountPlanned () {
```

Comment:
The function is not tested directly, but indirectly covered in payment-handler tests, why?

Reply:
Why not? https://github.com/commercetools/commercetools-adyen-integration/pull/283/commits/b30637e690c0d66bfd4f3ba43523bb090a5fd507